test/rubygems/test_gem_uninstaller.rb

```diff
@@ -149,6 +151,20 @@ def test_remove_not_in_home
     assert_path_exists @spec.gem_dir
   end
 
+  def test_remove_symlinked_gem_home
+    symlinked_gem_home = Tempfile.new("gem_home").path
+
+    FileUtils.ln_sf(@gemhome, symlinked_gem_home)
```

Comment:
I imagine at this point the file created by Tempfile is still open, and this line tries to remove it and replace it with a symlink. This works on unix but not on windows.

My suggestion is to create a temporary directory instead of a temporary file and use `ln_s` instead of `ln_sf` to create a symlink within that directory.